source/_integrations/envisalink.markdown

```diff
@@ -113,6 +114,10 @@ panic_type:
   required: false
   default: Police
   type: string
+unique_id:
```

Comment:
As per comment on the upstream PR; we no longer allow YAML configuration changes to integrations that connect to a device or service.